entity-service-impl/src/main/java/org/hypertrace/entity/service/util/DocStoreConverter.java

```diff
@@ -192,24 +211,36 @@ private static Filter createEqFilterForAttributeValue(String fieldName, Attribut
     Filter f = new Filter();
     f.setFieldName(fieldName);
     f.setOp(Op.EQ);
+    f.setValue(prepareRhsValueForSpecialValueListCase(attributeValue));
+    // Set child filters to empty array
+    f.setChildFilters(new Filter[]{});
+    return f;
+  }
+
+  private static Filter createNeqFilterForAttributeValue(String fieldName, AttributeValue attributeValue) {
+    Filter f = new Filter();
+    f.setFieldName(fieldName);
+    f.setOp(Op.NEQ);
```

Comment:
Do we support NEQ operator in Mongo? I don't think we do

Reply:
https://github.com/hypertrace/document-store/blob/main/document-store/src/main/java/org/hypertrace/core/documentstore/mongo/MongoCollection.java#L364

Reply:
Yes, I was coming to that only. First, we have to implement NEQ. You had pending PR.

Reply:
@skjindal93 https://github.com/hypertrace/document-store/pull/20

Reply:
Yeah. I started the work, but then since there wasn't any requirement at that time, didn't finish it up. Feel free to use the make changes on the existing branch/PR :)

Reply:
we have merged the `NEQ` filter support PR for mongodb. So, this can go ahead. I have added integration test.